crates/common/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "seda-common"
-version = "0.2.0"
+version = "0.4.0"
```

Comment:
Do we want to start matching versions to the devnet version?
If so, this could be something like `0.5.0-dev1`

Reply:
Hmmm I don't think matching versions makes sense. Since they could be in very different development cycles, but that's my opinion.

Reply:
Gonna merge for now, and if we decide to do this I can do it in a future PR